Assets/Scripts/ThirdPersonCamera.cs

```diff
@@ -57,16 +68,18 @@ private void Start()
 
     private void FixedUpdate()
     {
-
-        if (_levelCleared)
+     
+        
+        if ((_levelCleared || (_enableIsometricView && _onOrbit)))
```

Comment:
Parentheses actually redundant here, since the && is more binding than the || (order of operations for logic).

However, I would personally keep the () around the && statement for readability.

The () around the entire logic statement is redundant.